barretenberg/cpp/src/barretenberg/flavor/goblin_ultra.hpp

```diff
@@ -458,15 +458,16 @@ class GoblinUltra {
                 this->w_l = commitments.w_l;
                 this->w_r = commitments.w_r;
                 this->w_o = commitments.w_o;
-                this->sorted_accum = commitments.sorted_accum;
                 this->w_4 = commitments.w_4;
+                this->sorted_accum = commitments.sorted_accum;
                 this->z_perm = commitments.z_perm;
                 this->z_lookup = commitments.z_lookup;
                 this->ecc_op_wire_1 = commitments.ecc_op_wire_1;
                 this->ecc_op_wire_2 = commitments.ecc_op_wire_2;
                 this->ecc_op_wire_3 = commitments.ecc_op_wire_3;
+                this->ecc_op_wire_4 = commitments.ecc_op_wire_4;
```

Comment:
we weren't using this function until now but added it (missing stuff) for completeness in an earlier PR, corrected it now